client/site-profiler/components/hosting-information/index.tsx

```diff
@@ -16,6 +18,27 @@ export default function HostingInformation( props: Props ) {
 	const aRecordIps = dns.filter( ( x ) => x.type === 'A' && x.ip );
 	const supportUrl = useHostingProviderURL( 'support', hostingProvider, urlData );
 
+	useEffect( () => {
+		hostingProvider &&
+			recordTracksEvent( 'calypso_site_profiler_hosting_information', {
+				is_cdn: hostingProvider?.is_cdn,
+				provider_slug: hostingProvider?.slug,
+				provider_name: hostingProvider?.name,
+			} );
+	}, [ hostingProvider ] );
+
+	useEffect( () => {
+		urlData &&
+			recordTracksEvent( 'calypso_site_profiler_hosting_information', {
```

Comment:
The event name here is duplicate with the one above, maybe we want to use a different name here?

Reply:
I moved those two blocks to the hook and gave two different event names.